### PR TITLE
[Wav2Vec2] Update blog posts with new audio feature and other 

### PR DESCRIPTION
**Pull request by patrickvonplaten**

### CODE REVIEW

File: fine-tune-xlsr-wav2vec2.md [hunk 1]
```diff
@@ -735,9 +751,10 @@ The training loss goes down and we can see that the WER on the test set also imp
 
 The resulting model of this notebook has been saved to [`patrickvonplaten/wav2vec2-large-xlsr-turkish-demo`](https://huggingface.co/patrickvonplaten/wav2vec2-large-xlsr-turkish-demo)
 
-As a final check, let\'s load the model and verify that it indeed has learned to transcribe Turkish speech.
+### Evaluation
 
-Let\'s first load the pretrained checkpoint.
+In the final part, we evaluate our fine-tuned model on the test set and
```
Comment (patrickvonplaten) [inline fine-tune-xlsr-wav2vec2.md:756]:
```suggestion
In the final part, we run our model on some of the validation data to get a feeling for how well it works.
```

File: fine-tune-xlsr-wav2vec2.md [hunk 2]
```diff
@@ -733,22 +775,35 @@ trainer.train()
...
+model = Wav2Vec2ForCTC.from_pretrained(repo_name).to("cuda")
+processor = Wav2Vec2Processor.from_pretrained(repo_name)
 ```
 
 Now, we will just take the first example of the test set, run it through the model and take the `argmax(...)` of the logits to retrieve the predicted token ids.
 
 
 ```python
-input_dict = processor(common_voice_test["input_values"][0], return_tensors="pt", padding=True)
+input_dict = processor(common_voice_test["input_values"][0], sampling_rate=16_000, return_tensors="pt", padding=True)
```
Comment (osanseviero) [inline fine-tune-xlsr-wav2vec2.md:806]: Should this be 16000?

**Review by osanseviero — APPROVED** [review]
This is great! One for audio classification would be very nice, it has a ton of applications.

I left minor suggestions and fixes, but content-wise looks great!

File: fine-tune-xlsr-wav2vec2.md (hunk 2 above)
Comment (patrickvonplaten) [reply fine-tune-xlsr-wav2vec2.md:806]: Python allows `16_000` :-)

File: fine-tune-xlsr-wav2vec2.md [hunk 3]
```diff
@@ -646,17 +683,19 @@ To give more explanation on some of the parameters:
...
 from transformers import TrainingArguments
 
 training_args = TrainingArguments(
-  output_dir="/content/gdrive/MyDrive/wav2vec2-large-xlsr-turkish-demo",
-  # output_dir="./wav2vec2-large-xlsr-turkish-demo",
+  output_dir=repo_name,
   group_by_length=True,
   per_device_train_batch_size=16,
-  gradient_accumulation_steps=2,
+	gradient_accumulation_steps=2,
```
[the remark this answers is not in the document]
Comment (patrickvonplaten) [reply fine-tune-xlsr-wav2vec2.md:697]: Thanks!

**Comment by patrickvonplaten** [comment]
Thanks for the review!
